content/docs/reference/design-docs/train-sim-v3/driver-behavior-modules.en.md

```diff
@@ -138,21 +186,8 @@ To understand how this algorithm is designed, we need to consider two example ca
 - For step 1 and 2: if a neutral zone and a breaking instruction overlap, both are most constraining to different state properties: the neutral zone affects pantograph state, and the breaking instruction affects speed. The final state has to be a combination of both.
 - For step 3: We need to truncate integration steps to avoid overshoots, and thus avoid the neeed for feedback loops. Idealy, we want to truncate to the exact overshoot location. This overshoot location is not the same as the initial `dt` for the overshot constraint.
 
+### Should `truncate_integration_step` depend on the driver behavior module?
 
-#### Conclusion
-
-We decided to model driver behavior modules as follows:
-- numerical integration is performed within driver behavior modules
-- each driver behavior module reacts to a specific driving instruction
-- each driver behavior module is created ahead of time with:
-  - a slowdown coefficient
-  - a specific driving instruction
-  - path properties
-  - a simulator object
-- when queried:
-  - they take the current train state
-  - they return a new train state (including time, possibly smaller than `input_state.time + dt`)
-- driver behavior modules are queried for each driving instruction, and the decision with the lowest
-  acceleration is choosen
-- this decision is submitted for truncation to all driver behavior modules. The final decision is the one
-  with the smallest `dt`
```

Comment:
"The final decision is the one with the smallest `dt`"

AFAIU this isn't what we documented in the new "loop" section, I suppose this old section was outdated or confusing? (Just checking we're not missing something here)

Reply:
I think its a paragraph that was written before the last design decisions. Do you have any idea about this @multun or @woshilapin ?